vrk-kpa/suomidigi
# Updated acorn and minimist node modules.

To test: 
- `make build-theme`
- Check the build theme output for errors and check that the actual theme building works. F.e. No errors between 
```
> suomidigi@1.0.0 gulp /usr/src/app/public/themes/custom/suomidigi
> gulp "development"
```
and 
```
[07:34:51] Finished 'development' after 1.17 s
```
